src/devices/Nmea0183/Route.cs

```diff
@@ -0,0 +1,252 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Iot.Device.Common;
+
+namespace Iot.Device.Nmea0183
+{
+    /// <summary>
+    /// High-level representation of a route (a list of waypoints)
+    /// </summary>
+    public sealed class Route
```

Comment:
Any good reason to seal it?

Reply:
It's not really designed for inheritance. A list of something should normally be aggregated, not inherited from.